calculator.js

```diff
@@ -1,27 +1,119 @@
-
 const exampleAdditionInput = {
   num1: 3,
   num2: 5,
   operation: 'add',
 }
 
+const exampleAdditionSymbolInput = {
+  num1: 3,
+  num2: 5,
+  operation: '+',
+}
+
+const exampleSubtractionInput = {
+  num1: 3,
+  num2: 5,
+  operation: 'subtract',
+}
+
+const exampleSubtractionSymbolInput = {
+  num1: 3,
+  num2: 5,
+  operation: '-',
+}
+
+const exampleMultiplicationInput = {
+  num1: 3,
+  num2: 5,
+  operation: 'multiply',
+}
+
+const exampleMultiplicationSymbolInput = {
+  num1: 3,
+  num2: 5,
+  operation: '*',
+}
+
+const exampleDivisionInput = {
+  num1: 30,
+  num2: 5,
+  operation: 'divide',
+}
+
+const exampleDivisionSymbolInput = {
+  num1: 30,
+  num2: 5,
+  operation: '/',
+}
+
 const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+  // console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+
+  const num1 = Number.parseFloat(promptInput.num1);
+  const num2 = Number.parseFloat(promptInput.num2);
+
+  if (Number.isNaN(num1)) {
+    console.error(`${num1} is not a number.`);
+  } else if (Number.isNaN(num2)) {
+    console.error(`${num2} is not a number`);
+  }
```

Comment:
Consider `return`ing out of the function once you find invalid user input.

```suggestion
  if (Number.isNaN(num1)) {
    console.error(`${promptInput.num1} is not a number.`);
    return
  } else if (Number.isNaN(num2)) {
    console.error(`${promptInput.num2} is not a number`);
    return
  }
```